exp-player/addon/components/exp-exit-survey.js

```diff
@@ -95,6 +97,20 @@ export default ExpFrameBaseComponent.extend({
                     type: 'jsonschema',
                     description: 'JSON-schema defining second form.',
                     default: emailOptOutSchema
+                },
+                idealSessionsCompleted: {
```

Comment:
This frame fulfills a very specific use case. We are ignoring the general case for now.